pages/common/hive.md

```diff
@@ -0,0 +1,20 @@
+# hive
+
+> A CLI tools to use Apache Hive.
+> More information: <https://cwiki.apache.org/confluence/display/Hive/LanguageManual+Cli>.
+
+- Call a Hive interactive shell:
```

Comment:
```suggestion
- Start a Hive interactive shell:
```

Reply:
I saw python.md.
https://github.com/tldr-pages/tldr/blob/master/pages/common/python.md

It uses `Call ... interactive shell` expression.
It's difficult 🤔

Reply:
I based my suggestion in our `node` page

Reply:
Thanks for your review

Reply:
I think `Start` does make more sense here.

Reply:
OK. I'll use `Start`.